image.py

```diff
@@ -0,0 +1,601 @@
+# flake8: noqa
```

Comment:
Add description. The script is probably copy-pasted from SCT? If so, mention this in the description